python/dask_cudf/dask_cudf/backends.py

```diff
@@ -37,20 +37,24 @@ def concat_cudf(
 
 try:
 
-    from dask.dataframe.methods import group_split, hash_df
+    from dask.dataframe.utils import group_split_dispatch, hash_object_dispatch
 
-    @hash_df.register(cudf.DataFrame)
-    def hash_df_cudf(dfs):
+    @hash_object_dispatch.register(cudf.DataFrame)
+    def hash_object_cudf(
+        dfs, index=None, encoding=None, hash_key=None, categorize=None
```

Comment:
Should we assert these values aren't set to something we don't yet support?

Reply:
Right - This is actually something I wanted to revisit/discuss...  These are the kwargs for [`pandas.util.hash_pandas_object`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.util.hash_pandas_object.html).   In dask, we only set the `index` argument and leave everything else as default (we always set `index=False`, which just means that we are not hashing the index with the columns).

Should we (1) only include `index`, or (2) include all kwargs and print a warning if any of the "unused" kwargs are specified?  For either of these cases, we can either (a) assert that `index==False`, or (b) implement both `index=True` and `index=False` versions (not difficult)